tasks/go.py

```diff
@@ -39,6 +42,31 @@ def get_deps(key):
 
     return deps.get(key, {})
 
+def process_deps(ctx, target, version, kind, step, verbose=False):
```

Comment:
We should probably extract the bootstraping logic from `go.py` at this point.

Reply:
I was thinking about that. Sure.